### ISSUE TITLE
bug: cannot scroll the job data tab

### ISSUE THREAD

**Issue by jaroslaw-weber** [felixmosh/bull-board#479]
how to reproduce:

- create a job with a lot of parameters

```
queue.add('jobname', {
a1: '1',
a2: '2',
...
a2000:'2000'
})
```

- open queue in bull board
- try to scroll to see all data
- it's impossible to scroll

**Comment by felixmosh** [comment]
Hi, which version of bull-board to u use?

**Comment by jaroslaw-weber** [comment]
4.6.0
it was working with 4.3.1

**Comment by jaroslaw-weber** [comment]
this may be related? : https://github.com/felixmosh/bull-board/pull/477

**Comment by felixmosh** [comment]
Yeap :|

**Comment by felixmosh** [comment]
fixed in `v4.6.1`

**Comment by jaroslaw-weber** [comment]
thanks for quick fix!! :)
